### ISSUE TITLE
Determine the best deployment strategy for the lambda fetchers

### ISSUE THREAD

**Issue by caparker** [openaq/openaq-fetch#920]
Created to discuss the best way to deploy the updated fetchers. 
A quick overview of how its set up to work

1. event triggers the scheduler lambda
2. scheduler lambda then determines which sources should be run and submits one SQS message that includes all the selected sources passed in the body of the message
3. Fetcher lambda is responding to SQS and looks in the event to find the sources (if available)
4. The fetcher then combines the source information with the env for the deployment and builds the file

So with this setup we will need a different instance of the lambda stack for each deployment that we want to run. That means deploying a different stack for each, with a different .env file for each, which doesnt seem sustainable in the long term but good for now in my opinion. 

Long term I am thinking we need to just have one deployment of the stack and that the scheduler pulls from the database to see what it should do.

**Comment by caparker** [comment]
And right now I think we have at least two deployments to manage

1. All sources, no offset, assuming that all can be finished in the 15min limit
2. AirNow, 24 hour offset

**Comment by russbiggs** [comment]
The two deployment option seems reasonable

**Comment by caparker** [comment]
Are you proposing to just keep the two deployment option long term? The downside of that is the need to manually deploy and not use github actions.

**Comment by russbiggs** [comment]
not long term, "good for now" as you said in the original comment

**Comment by majesticio** [comment]
I assume we have 3 lambda deployments now with Mexico/Sinaica? And we are considering using several in the future based off of geography and how much time an adapter needs?

**Comment by russbiggs** [comment]
Mexico/SINAICA is in the main deployment I believe. perhaps @caparker can confirm

**Comment by caparker** [comment]
Correct. We have 3 deployments of the lambda; the main, London and the air
now offset deployment. The Mexico data comes form the main one.
